.gitignore

```diff
@@ -0,0 +1,5 @@
+.gradle/
+.idea/
+aws-emrserverless-application/aws-emrserverless-application.zip
+aws-emrserverless-application/build/
+build
```

Comment:
Remove this gitignore file as it contains internal files/folders ref which are not present in this repo

Reply:
Removed.